maven-core/src/main/java/org/apache/maven/internal/MultilineMessageHelper.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.maven.internal;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Helper class to format multiline messages to the console
+ */
+public class MultilineMessageHelper
+{
+
+    private static final int DEFAULT_MAX_SIZE = 65;
+    private static final char BOX_CHAR = '*';
+
+    public static String separatorLine()
+    {
+        StringBuilder sb = new StringBuilder( DEFAULT_MAX_SIZE );
+        repeat( sb, '*', DEFAULT_MAX_SIZE );
+        return sb.toString();
+    }
+
+    public static List<String> format( String... lines )
+    {
+        int size = DEFAULT_MAX_SIZE;
+        int rem = size - 4; // 4 chars = 2 box_char + 2 spaces
```

Comment:
Could we just name this 'remainder'? I don't think we have a limit on variable names, so better make them clear and descriptive IMO.

Reply:
Done.